src/stories/Tabs.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { Tabs, Tab } from '../components/Tabs';
+
+const TabsExample = () => {
+  const [tab, setTab] = useState(0);
+
+  return (
+    <div>
+      <Tabs activeTab={tab}>
+        <Tab onClick={() => setTab(0)} tab={0}>
```

Comment:
Could we handle this in a way without manually setting the tab index somehow?

Reply:
Maybe, but I don't know what's the best way of doing it.

Perhaps we could provide an API such as

```javascript
<Tabs>
  <TabPane header="Tab 1">
    Lorem ipsum...
  </TabPane>
</Tabs>
```

in addition to the current one?